components/feature/logins/schemas/mozilla.components.feature.loginsexceptions.db.LoginExceptionDatabase/1.json

```diff
@@ -0,0 +1,40 @@
+{
+  "formatVersion": 1,
```

Comment:
It looks like we have to `LoginExceptionDatabase/1.json ` files 🤔

Reply:
Oops I moved/renamed the feature so I need to remove this one 😅

Reply:
👍